AirCasting/Map/Heatmap.swift

```diff
@@ -0,0 +1,344 @@
+// Created by Lunar on 10/11/2021.
+//
+
+import UIKit
```

Comment:
I think that you can delete this one and leave Foundation only. If not please do otherwise :)